_includes/manuals/1.10/4.4.3_prov_templates.md

```diff
@@ -40,3 +40,10 @@ The final entry, Operating System default, can be set by editing the Operating S
 ##### Associating an Operating System default template
 
 You will need to associate at least one PXELinux, Provision, and Finish template to your Operating System, and this must be done in two steps. First edit each of the templates, switch to the Association tab, and ensure the appropriate OSs are checked. Then edit the Operating System, switch to the Templates tab, and choose a default template for each template kind.
+
+##### Templates Details
+###### Finish Templates
+Finish templates are one of two methods to customize and finish the installation for image based installs, the other alternative being user_data templates. In contrast to user_data they are available for all hypervisors that support image based installs where the foreman server can reach the newly installed machine via ssh and scp. The script generated from the finish template is copied by the Foreman to the newly installed system via scp using username and password specified with the image. It is then executed by connecting again via ssh, making the script executable and either executing it directly or via sudo if the specified user is not root. Standard output and standard error are logged to a file in the same directory named *bootstrap-UUID.log*.
+
+###### User Data Templates
+User Data templates are one of two methods to customize and finish the installation for image based installs, the other alternative finish templats. In this case the installed machine must not be reachable via ssh by the Foreman server but the machine must be able to reach the host via http(s).
```

Comment:
maybe it makes more sense to move the "the other alternative" parts to a very short general introduction in "Templates Details"